pom.xml

```diff
@@ -157,6 +157,7 @@
         <maven-enforcer-plugin.version>1.3.1</maven-enforcer-plugin.version>
 
         <hadoop.version>2.7.3</hadoop.version>
+        <hadoop.aliyun.version>3.0.0-alpha2</hadoop.aliyun.version>
```

Comment:
I find the latest version is 3.0.0

Reply:
For now, ODPS are depended on 3.0.0-alpha2.